### PR TITLE
fix(plugin-chart-echarts): fix GraphViz crash when DnD FF enable

### PR DESCRIPTION
**Pull request by zhaoyongjie**
## Summary
fix graph viz crash when dnd FF enable

## use select control in Graph Viz
image

## use DnD control in Graph Viz
image

### CODE REVIEW

File: plugins/plugin-chart-echarts/src/Graph/controlPanel.tsx [hunk 1]
```diff
@@ -28,22 +28,20 @@ import {
 } from '../controls';
 
 const noopControl = { name: 'noop', config: { type: '', renderTrigger: true } };
-const columnSelectControl = {
+
+const requiredEntity = {
+  ...sharedControls.entity,
+  clearable: false,
+  // do not export private name 'filterOption'
+  filterOption: sharedControls.entity,
```
Comment (ktmud) [inline plugins/plugin-chart-echarts/src/Graph/controlPanel.tsx:36]: Shouldn't `filterOption` be a function? I didn't get what the comment `do not export private name 'filterOption'` means. Are you just trying to disable the type-to-search filter?

Comment (zhaoyongjie) [reply plugins/plugin-chart-echarts/src/Graph/controlPanel.tsx:36]: Hi, @ktmud
typescript raise a type error `TS4082: Default export of the module has or is using private name 'FilterOption'.`, so I guess `filterOption` is a private name.

image

Comment (zhaoyongjie) [reply plugins/plugin-chart-echarts/src/Graph/controlPanel.tsx]: thanks for this suggestion. I have changed with this suggestion. Unfortunately, The main repo also needs some repairs.

**Review by ktmud — APPROVED** [review]
LGTM

**Pull request merged**